pbjs-genfiles/README.md

```diff
@@ -0,0 +1,5 @@
+This folder will contain JavaScript files and TypeScript type definitions
```

Comment:
`will contain` or `contains`? Present or future tense?

Reply:
I don't know if it should be present or future :) It's empty in the repository, but will be populated with some auto-generated files after someone runs `npm install`. What do you think?

Reply:
Maybe something like `This folder is used for auto-generated JavaScript files and TypeScript type definitions`.

I'm nit picking, its not going to confuse many.